o3po/includes/class-o3po-primary-publication-type.php

```diff
@@ -1505,6 +1513,17 @@ public function get_the_content( $content ) {
     }
 
 
+    public function scirate_content($scirate_url_abs_prefix, $post_id, $post_type) {
+            if(!empty($scirate_url_abs_prefix)) {
+                return '<tr><td>Scirate:</td><td><a href="' 
+                         . esc_attr($scirate_url_abs_prefix
+                         . get_post_meta( $post_id, $post_type . '_eprint', true ) ) . '">' 
+                         . esc_html($scirate_url_abs_prefix
+                         . get_post_meta( $post_id, $post_type . '_eprint', true )) . '</a></td></tr>';
+            }
+            return '';
+    }
+ 
```

Comment:
As said above, the function seems overkill as the same effect can be achieved by adding a single line. I would appreciate if you could simplify this PR by removing this function.